src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,30 @@
-export const PostInfo = () => (
-  <>Put the post here</>
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">
+        {post.title}
+      </h3>
+
+      <UserInfo user={post.user} />
```

Comment:
Theoretically `post.user` could be `null`:

![image](https://github.com/mate-academy/react_static-list-of-posts-js/assets/86923089/75ab64d2-2f60-4fc9-86f0-9f4a0fb9e107)

In this case, we will get an error (we try to read `email` and `name` from `null` in the UserInfo component).
So we should handle it - do not render `UserInfo` at all  if `post.user` is not defined. use [conditional rendering](https://react.dev/learn/conditional-rendering#logical-and-operator-)